picard/options.py

```diff
@@ -436,3 +437,10 @@ def make_default_toolbar_layout():
 
 def init_options():
     pass
+
+
+def get_option_title(text):
+    key = ('setting', text)
+    if key not in Option.registry or not Option.registry[key].title:
```

Comment:
If no key, I think we should return `None` (or even raise an exception and catch it in the caller?).
When there's no title, I would rather display the option name (something like "No title for setting <name>").

```python

def get_option_title(name):
    key = ('setting', name)
    if key not in Option.registry:
        return None
    title = Option.registry[key].title
    if title:
        return title
    return N_("No title for setting '%s'") % name
```

Minor thing, but option name (rather than text).

Reply:
I like it.  Updated in https://github.com/metabrainz/picard/pull/2636/commits/648556d072d221626577346247a35c2986a8bb5d